linters/bandit.js

```diff
@@ -53,7 +53,24 @@ module.exports = class Bandit {
    * @param {Buffer} data The raw linter results data
    */
   parseResults (data) {
-    return JSON.parse(data)
+    let parsedData = JSON.parse(data)
+
+    if (parsedData.errors.length !== 0) {
```

Comment:
Be careful with if != checks on lengths.  Some functions may return a -1 to indicate conditions.  As a result ever time I see a check like this, alarm bells go off in my head.  :)

Reply:
Thank you, that's good advice and a reminder.